notify/notify.go

```diff
@@ -284,6 +285,11 @@ func NewMetrics(r prometheus.Registerer, ff featurecontrol.Flagger) *Metrics {
 			Name:      "notification_requests_failed_total",
 			Help:      "The total number of failed notification requests.",
 		}, labels),
+		numNotificationSuppressedTotal: prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
If we decide to follow the other comment, we could add a `reason` label here. Then, users can split up the individual rates based on what did the suppression (was it silenced, or part of a mute time? etc).

It may be worth adding this even if we don't follow my other comment - one reason for active times, and another for mute times, in case users are using both simultaneously.

Reply:
Like I said in response to the other comment, I can add the metrics to the other stages as well. And agreed, I think a `reason` label is much more obviously valuable now